front/app/containers/Navbar/components/NotificationMenu/components/Notification/index.tsx

```diff
@@ -290,14 +289,13 @@ export default class Notification extends PureComponent<Props> {
             }
           />
         );
-      case 'inappropriate_content_flagged':
+      default:
```

Comment:
So we're explicitely not mentioning the type of the notification here, because it has to be specified in the module. Here, we will use the Outlet to render any notification that's not by the `switch case`s in this file.